src/System.Security.Permissions/tests/project.json

```diff
@@ -0,0 +1,29 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.2-beta-devapi-24415-01",
+    "System.Collections": "4.0.12-beta-devapi-24415-01",
+    "System.Console": "4.0.1-beta-devapi-24415-01",
+    "System.Globalization": "4.0.12-beta-devapi-24415-01",
+    "System.Linq.Expressions": "4.1.1-beta-devapi-24415-01",
+    "System.ObjectModel": "4.0.13-beta-devapi-24415-01",
+    "System.Runtime": "4.2.0-beta-devapi-24415-01",
+    "System.Runtime.Extensions": "4.1.1-beta-devapi-24415-01",
+    "System.Text.RegularExpressions": "4.2.0-beta-devapi-24415-01",
+    "System.Threading.Tasks": "4.0.12-beta-devapi-24415-01",
+    "System.Security.AccessControl": "4.0.0",
+    "System.Security.Cryptography.Primitives": "4.0.0",
+    "System.Security.Cryptography.X509Certificates": "4.1.0",
+    "test-runtime": {
+      "target": "project",
+      "exclude": "compile"
+    },
+    "Microsoft.xunit.netcore.extensions": "1.0.0-prerelease-00508-01",
+    "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00520-02"
```

Comment:
Should be '1.0.0-prerelease-00704-03'

00:01:07.350 /mnt/resource/j/workspace/dotnet_corefx/dev_api/centos7.1_debug_prtest/build.proj(111,5): error : Dependency validation error: for Microsoft.DotNet.BuildTools.TestSuite 1.0.0-prerelease-00520-02 in /mnt/resource/j/workspace/dotnet_corefx/dev_api/centos7.1_debug_prtest/src/System.Security.Permissions/tests/project.json package version is '1.0.0-prerelease-00520-02' but expected '1.0.0-prerelease-00704-03' for packages matching '^(?i)Microsoft\.DotNet\.BuildTools\.TestSuite$'